distribution/lib/Standard/Base/0.0.0-dev/src/Meta.enso

```diff
@@ -39,31 +39,37 @@ type Atom
 
        Returns a constructor value of the given atom.
     constructor : Constructor
-    constructor self = get_atom_constructor self.value ...
+    constructor self =
+        java_constructor = get_atom_constructor self.value ...
+        Meta.Constructor.Value java_constructor
```

Comment:
```suggestion
        Meta.Constructor.Value (java_constructor ...)
```

Looking at @kustosz's messages on Discord, I'd expect this. But somehow I assume what you have works. Any idea why here we _don't_ need `...`?

Reply:
Having an AtomConstructor in a local variable and then using the local variable doesn't seem to trigger the re-evaluation.
As it worked went with it!

Reply:
Sure! Just asking to try to understand this better. Glad that this fixes our issue.